server/src/main/resources/application.properties

```diff
@@ -18,8 +18,9 @@ conductor.db.type=memory
 
 conductor.indexing.enabled=false
 
-#Dynomite Cluster details.
```

Comment:
Changed this comment because these properties apply not only to Dynomite. They apply to other Redis configurations.